<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>508</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DENAS Simulator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/overlay/denasButtonless.svg</normalon>
   </iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/*(2)*/
color: rgb(32, 74, 135);
background-color: rgb(32, 74, 135);

/*
(4)
background-color: #616E7C;
color: 
*/</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mainWindowCentralQWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainWindowQHBoxLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="controls" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>186</width>
          <height>488</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>186</width>
          <height>488</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>186</width>
          <height>488</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="powerButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>430</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/powerButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="downButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>260</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>40</horstretch>
           <verstretch>40</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/downButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>200</number>
         </property>
        </widget>
        <widget class="QPushButton" name="menuButton">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/menuButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="backButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/backButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="leftButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/leftButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>150</number>
         </property>
        </widget>
        <widget class="QPushButton" name="upButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>160</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/upButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>200</number>
         </property>
        </widget>
        <widget class="QPushButton" name="rightButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>210</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/rightButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>150</number>
         </property>
        </widget>
        <widget class="QPushButton" name="okButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>210</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/okButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="battery">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>100</number>
         </property>
        </widget>
        <zorder>menuButton</zorder>
        <zorder>downButton</zorder>
        <zorder>backButton</zorder>
        <zorder>okButton</zorder>
        <zorder>powerButton</zorder>
        <zorder>upButton</zorder>
        <zorder>leftButton</zorder>
        <zorder>rightButton</zorder>
        <zorder>battery</zorder>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Noto Sans Adlam&quot;;</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="menuTab">
         <attribute name="title">
          <string>Menu</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sessionTab">
         <attribute name="title">
          <string>Session</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
